src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.tsx

```diff
@@ -128,25 +131,41 @@ export const BreachDetailsView = (props: Props) => {
         </h2>
         <div className={styles.content}>
           <p>
-            {l10n.getString("breach-overview-new", {
-              breachDate: (breach.BreachDate as unknown as Date).toLocaleString(
-                getLocale(l10n),
-                {
+            {breach.Name === "BVD" && getLocale(l10n).split("-")[0] === "en" ? (
+              <>
+                In approximately August 2021, hundreds of gigabytes of business
+                data collated from public sources was obtained and later
+                published to a popular hacking forum. Sourced from a customer of
+                Bureau van Dijk&apos;s (BvD) &quot;Orbis&quot; product, the
+                corpus of data released contained hundreds of millions of lines
+                about corporations and individuals, including personal
+                information such as names and dates of birth. The data also
+                included 28M unique email addresses along with physical
+                addresses (presumedly corporate locations), phone numbers and
+                job titles. There was no unauthorised access to BvD&apos;s
+                systems, nor did the incident expose any of their or parent
+                company&apos;s Moody&apos;s clients.
```

Comment:
Ah, apologies, I think I misunderstood this copy&hellip;
This copy seems to be a _slightly_ modified version of the HIBP breach Description. Not sure if we keep a copy of the `Description` from HIBP in our DB (although [this](https://github.com/mozilla/blurts-server/blob/ac6efb9ef3db288048e91f66cd4ebc6a876b60fa/src/db/tables/breaches.js#L62) implies we do) &mdash; but what if we just maintain an array of breaches where we revert back to the HIBP `.Description` and remove/sanitize any HTML markup. Something like https://runkit.com/pdehaan/65f33e17179ddd0008eb640c

This would mean we don't need to hardcode strings into our JSX and have potentially complex nested inlined terneries. Although not 100% certain whether we potentially `sanitize` here, or in the breaches table when we add breach data.

PROS: Less hardcoding.
CONS: Possibly 2% more logic and somewhat less flexibility if we want to diverge from what we get from HIBP.

Reply:
👍 One or two exceptions are fine &mdash; we can reconsider refactoring if we need 3+ overrides. Considering this is the first one in 5+ years, I think we're safe. And HIBP changing descriptions/data is pretty rare&hellip; not zero, but rare.